packages/component/src/lib/embedding_view/EmbeddingViewMosaic.svelte

```diff
@@ -146,9 +146,9 @@
         return;
       }
       let captuerd = tooltip;
-      effectiveTooltip = captuerd.value;
+      effectiveTooltip = (captuerd.valueFor(client) ?? null) as any;
       let listener = () => {
-        effectiveTooltip = captuerd.value;
+        effectiveTooltip = (captuerd.valueFor(client) ?? null) as any;
```

Comment:
Mosaic's `valueFor` returns `unknown`, so we have to somehow assert the type here, using `any` for simplicity.